### ISSUE TITLE
1.11.2

### ISSUE THREAD

**Issue by TheFluffyDragon** [jaredlll08/CraftTweaker#248]
Issue Description:
Changes to vanilla recipes do not work.
/mt reload and a relaunch doesn't fix the issue

What happens: 
After the script is added it fails to work without giving any errors as well as not showing the recipe update withing JEI

What you expected to happen:
The recipe to work.

Script used:
recipes.addShaped(<minecraft:ice>), [[<minecraft:packed_ice>, <null>, <null>], 
[<null>, <null>, <null>], [<null>, <null>, <null>]]);

Minetweaker.log file:

[POSTINITIALIZATION][CLIENT][ERROR] system: JEI is already defined in that package
[AVAILABLE][CLIENT][INFO] MineTweaker: Building registry
[AVAILABLE][CLIENT][INFO] MineTweaker: Sucessfully built item registry
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding command minetweaker
[SERVER_STARTED][SERVER][INFO] tried to remove command: minetweaker failed. THIS IS NOT AN ERROR!

Affected Versions:

- Minecraft: 1.11.2
- Forge: 13.20.1.2393
- Crafttweaker: 1.11.2-3.0.26

-Mods 
https://pastebin.com/9dUB9xsJ

**Comment by jaredlll08** [comment]
Use pastebin for your scripts.
Where did you put your scripts?

**Comment by TheFluffyDragon** [comment]
I put the script in C:\Users\---\AppData\Roaming\.minecraft\scripts

https://pastebin.com/2expDKmi that's the script

**Comment by jaredlll08** [comment]
Where are your mods located?

**Comment by TheFluffyDragon** [comment]
C:\Users\---\AppData\Roaming\.minecraft\mods

The same as any other custom setup, I thought it was a problem with configs so i deleted minecraft and reinstalled it, problem was still there so that's when I posted here.

**Comment by jaredlll08** [comment]
What is your script file called?

**Comment by TheFluffyDragon** [comment]
Skyblock.zs

The error seems to be with recipes.addShaped, after rewatching your video on how to balance a mod pack, i changed only Shaped to Shapeless and it now works, tho when I change it back to Shaped it then fails

**Comment by jaredlll08** [comment]
Upload your log to pastebin. As well as your scripts.
And read the wiki.

**Comment by TheFluffyDragon** [comment]
https://pastebin.com/E2eDHA9E

I deleted the old recipe after the shapeless one worked, after reloading the script I went to try shaped again and it now works.

After adding the old script listed above in the paste bin I now get this error in chat. So it's something within that script that is causing the problems, I'm not sure why it was not giving a error output before.

ERROR: Error parsing Skyblock.zs:13 -- ; expected Scripts reloaded
